### DISCUSSION TITLE
코드 수정사항 체크리스트입니다.

### DISCUSSION THREAD

**Discussion by hoospacekor** [7-7-2/DDocker_FE#75] [Ideas]
- [x] @cuconveniencestore 님 api함수 코드내에 `try-catch` 혹은 `.catch()`  예외처리 추가해주세용 @swon95 상원님도 PR전에 안 작성하셨으면 참고하셔서 작성해주세용 (**저는 1번이 가독성이 좋아서 좋습니다**) 근데 쓰고보니 getUserId쪽에서 처리를 해주는 게 더 적합해 보이네요 아무튼 예외처리 부탁드립니다!! 확인하시고 여기 체크해주세용
```javascript
//1
export const getUserDocRef = async () => {
  const userId = (await getUserId().catch(e=>console.error(e))) as string;
  return doc(getFirestore(), Collections.USERS, userId);
}; 

//2
export const getUserDocRef = async () => {
  try {
    const userId = (await getUserId()) as string;
    return doc(getFirestore(), Collections.USERS, userId);      
  } catch (error) {
   return console.log(error)
  }
};
```

- [x] 상원님 금요일에 말씀드렸던 useStorage api코드 네이밍 수정부탁드립니다. 확인하시고 수정하셨으면 마찬가지로 체크박스 눌러주세용

- [x] CoffeeOptionSelection은 현재로써는 너무 장황해보입니다. 향후 커스텀 훅을 통한 축약 혹은 셀렉터를 활용한 계산 위임을 고민해보면 훨씬 좋은 컴포넌트로 다듬을 수 있을 것 같습니다. 급한 게 아니라 리팩토링 시 같이 고민해보면 좋을 사항이라 미리 작성해두겠습니다.😁😁

- [x] `<PinturaEditModal />` props 및 내부 콜백함수가 장황합니다. 깔끔한 JSX를 위해....개선하겠습니다!!

**Comment by wavescats** [comment]
확인했습니다 😎✔

**Comment by cuconveniencestore** [comment]
@hookor 님 말씀대로 CoffeeOptionSelection 부분은 작성할때 옵션 부분 방식 확정이 되어있지 않아 현재의 상태로 작성해 두었고, 추후 리팩토링 진행할 예정이었습니다! 감사합니다! 🔌
